controller/instance_manager_controller.go

```diff
@@ -473,16 +505,38 @@ func (imc *InstanceManagerController) syncInstanceManagerPDB(im *longhorn.Instan
 		if imPDB == nil {
 			return nil
 		}
+
+		imc.logger.Debugf("Node %v is marked unschedulable, try to remove %v PDB", imc.controllerID, im.Name)
 		canDelete, err := imc.canDeleteInstanceManagerPDB(im)
 		if err != nil {
 			return err
 		}
+
 		if !canDelete {
 			return nil
 		}
 		return imc.deleteInstanceManagerPDB(im)
 	}
 
+	// CA considers a node is unremovable when the node has pod that does not allow
+	// for disruptions. When this setting is enabled Longhorn will actively delete
+	// IM PDB when volume is detached, leaving single replica IM PDB to protect volume
+	// data.
```

Comment:
If the setting is enabled, Longhorn needs to retain as least IM PDBs as possible so that there is at least one replica for each volume while no redundant PDB blocking the CA draining.
If the setting is disabled, Longhorn will blindly create IM PDBs. When a node is being drained/cordoned while the draining won't crash volumes, the PDBs on the node will be cleaned up

Reply:
Reword.